lib/services/plugins-service.ts

```diff
@@ -350,6 +373,10 @@ export class PluginsService implements IPluginsService {
 		}).future<string>()();
 	}
 
+	private normalizeMinCordovaVersion(version: string): string {
+		return _.startsWith(version, '>=') ? version.substr(2) : version;
```

Comment:
a bit of [research](http://cordova.apache.org/docs/en/3.0.0/plugin_ref_spec.md) leads me to believe that a more complex approach is indeed needed maybe we should move the [semver](https://github.com/npm/node-semver) module to common and use it here?